scripts/cibuild/ConfigureVstsBuild.ps1

```diff
@@ -42,7 +42,7 @@ Function Get-Version {
         # Generate the new minor version: 4.0.0 => 40000, 4.11.5 => 41105. 
         # This assumes we only get to NuGet major/minor 99 at worst, otherwise the logic breaks. 
         #The final version for NuGet 4.0.0, build number 3128 would be 15.0.40000.3128
-        $finalVersion = "15.0.$((-join ($ProductVersion -split '\.' | %{ '{0:D2}' -f ($_ -as [int]) } )).TrimStart("0")).$build"    
+        $finalVersion = "16.0.$((-join ($ProductVersion -split '\.' | %{ '{0:D2}' -f ($_ -as [int]) } )).TrimStart("0")).$build"    
```

Comment:
your PR comment says no manual action needed any more, but we would every minor VS release, no? Is there something we can do to get the "16" from NuGet's major version + 11 and get "0" from NuGet's minor version?

Reply:
We would need to change 15 to 16 in the ConfigureVstsBuild.ps1 script.
One thing is powershell and the other one is msbuild. 

I'm sure if we dig some more we can come up with something better.

Reply:
oh and we don't need to update every minor  VS release. 
That's factored in the 3rd part of the version. 

The 2nd part is always 0 in case we screw up the versioning :)
So: 
$(VSMajor).0.$(NuGetMajor)$(NuGetMinor)0$(NuGetPatch)0.$(Build)